src/windows/MediaProxy.js

```diff
@@ -26,6 +26,15 @@ var MediaError = require('cordova-plugin-media.MediaError');
 
 var recordedFile;
 
+function setTemporaryFsByDefault(src) {
+    if (!/.+:/.test(src)) {
```

Comment:
This test seems like it could collide with absolute file paths (e.g., "c:\foo.bar") or UNC paths (e.g., "\\?\C:\foo.bar").  Are you sure this is adequate?  I'm curious about whether creating a new Windows.Foundation.Uri would be more likely to detect such sources.